Source/Urho3D/IO/Log.cpp

```diff
@@ -135,6 +136,21 @@ void Log::SetQuiet(bool quiet)
     quiet_ = quiet;
 }
 
+void Log::WriteFormat(int level, const char* format, ...)
+{
+    if (!logInstance || logInstance->level_ > level)
+        return;
+
+    // forward to normal write after formatting the input

```

Comment:
With a capital letter

Reply:
are you referring to the comment? i.e. Write vs write?

Reply:
forward -> Forward